walkingastro.lic

```diff
@@ -76,8 +76,18 @@ class WalkingAstro
     waitrt?
   end
 
+  def can_see_sky?
+    # If we didn't see the sky last time we checked
+    # and we're in the same room still then don't redundantly check
+    # so that we don't spam the game window with "That's a bit hard to do while inside".
+    return false if @last_room_check == DRRoom.title && @last_sky_check == false
+    @last_sky_check = DRC.can_see_sky?
+    @last_room_check = DRRoom.title
+    return @last_sky_check
+  end
```

Comment:
Using the room title instead of the room id because not all rooms have ids, such as in a vault or maze.